.github/actions/setup-build/action.yml

```diff
@@ -0,0 +1,14 @@
+name: "Setup Gradle"
+description: "Setup Gradle"
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-java@v2
+      with:
+        java-version: '17'
```

Comment:
I understand you have refacored this from the other pipeline.  

However, just double-checking: In `README.md` (in root) the minimum version of `JDK 11` is mentioned. Is it safe to use `17` here?

Reply:
Yes, see https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/pull/934